app/Models/RoleNode.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Model;
+
+class RoleNode extends Model
```

Comment:
This is only a Pivot table, correct? Then this is more correct:

```suggestion
class RoleNode extends Pivot
```

Reply:
I believe he followed what we'v had for MountNode

https://github.com/pelican-dev/panel/blob/98d8510f11f3d007bb46b57519a09595a8216e22/app/Models/MountNode.php#L7
https://github.com/pelican-dev/panel/blob/98d8510f11f3d007bb46b57519a09595a8216e22/app/Models/Node.php#L242-L245